04 - Reserving/Data Simulation/_scripts/feature_generation.py

```diff
@@ -0,0 +1,182 @@
+def feature_generation(V,lob,inflation,seed1):
+
+### Definition of the function Feature.Generation that generates features ###
+#############################################################################
+
+### Define the function Feature.Generation
+### We have the following inputs:
+### V = totally expected number of claims
+### LoB.dist = categorical distribution for the allocation of the claims to the 4 lines of business
+### inflation = growth parameters (per LoB) for the numbers of claims in the 12 accident years
+### seed1 = set the seed for reproducibility
+
+    import numpy as np
+    import pandas as pd
+    import scipy as sp
+    from scipy import stats
+    import math
+    import random
```

Comment:
In standard cases we want to ensure imports are at the top of a Python script/module, outside of any functions